### DISCUSSION TITLE
New file detection for language qualifier: Horse64

### DISCUSSION THREAD

**Discussion by MichaelHinrichs** [community/community#123024] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Horse64 programing language: https://horse64.org/
It uses the file extension ".h64".

**Comment by LiteBrite82** [comment]
@MichaelHinrichs 
Thanks for highlighting new tech! Can you elaborate a little more on your post? Is this related to GitHub Code Search, or, is this just general information? 

(Its great community participation either way, I'm just looking for context so it can be categorized appropriately)

**Reply by MichaelHinrichs** [reply]
Yeah, i would like it to be added to the searchable languages.
![Screenshot 2024-05-10 104819](https://github.com/community/community/assets/37460517/9e8c6dca-931a-48cb-b805-92a834b765d3)
